roll/websockets.txt

```diff
@@ -0,0 +1,69 @@
+Websockets inner and outer workings
```

Comment:
This documentation is valuable, is there a reference page where it came from? Can we set/reuse something alike `http.HTTPStatus` to deal with codes and error messages in a more user-friendly way? (like when we raise errors, I'm pretty sure I'll prefer to receive the extended version of the error 😉)

Reply:
It's from the RFC: https://tools.ietf.org/html/rfc6455
When i was fiddling with the websocket flow, i kep that .txt as a reminder, it should be gone or pushed into a docs/. I kept it here for you, guys, as a quick reference.

Reply:
Thanks! Too bad it's not a bit more [verbose on the websockets side](https://github.com/aaugustin/websockets/blob/f83362ef91a6ac2ea89fd8bda4d18ff22ffb7a40/websockets/exceptions.py#L118) for easy reuse, that's a “just” a PR-away though.